### PR TITLE
Add global styles

### PR DESCRIPTION
**Pull request by Johannafendrich**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -9,10 +9,6 @@
       name="description"
```
Comment (steinberggit) [inline client/public/index.html:9]: You could change the name to 'my-homie'

File: client/public/index.html [hunk 2]
```diff
@@ -9,10 +9,6 @@
...
       content="Web site created using create-react-app"
```
Comment (steinberggit) [inline client/public/index.html:10]: ... and could change the description

Comment (Johannafendrich) [reply client/public/index.html:10]: Perfect, thank you for pointing out that I already wanted to do that. 😅

File: client/public/index.html (hunk 1 above)
Comment (Johannafendrich) [reply client/public/index.html:9]: I agree with you  👌🏼

**Pull request merged**